.github/workflows/upload-docs-ios.yml

```diff
@@ -0,0 +1,43 @@
+name: Upload Docs iOS
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+      - 'test-doc-*'
+
+jobs:
+  upload-docs:
+
+    runs-on: macos-latest
+
+    steps:
+    - name: Checkout current branch
+      uses: actions/checkout@v4
+    - name: Install Jazzy
+      run: |
+        gem install jazzy
+    - name: Generate docs
+      run: |
+        jazzy
+    - name: Checkout docs branch
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        ref: docs
+        clean: false
```

Comment:
Looks like this command doesn't work as expected. 

In the case of updating the existing doc the fetching of the `docs` branch invalidates local changes. So there is nothing to commit after that. 

It does work when the `docs` brach doesn't have the `/docs` folder. But once the folder is there: 
```
Your branch is up to date with 'origin/docs'.
nothing to commit, working tree clean
```

We need to fix it; The docs should be updated automatically. 

As an alternative, we can try to merge the `release-X.X.X` branch into `docs` and build the docs right there.